extras/indexingExample/src/main/java/RyaClientExample.java

```diff
@@ -132,18 +133,17 @@ public static void main(final String[] args) throws Exception {
                      "}";
 
             // Load some statements into the Rya instance.
-            final AccumuloRdfConfiguration conf = new AccumuloRdfConfiguration();
-            conf.setTablePrefix(ryaInstanceName);
-            conf.set(ConfigUtils.CLOUDBASE_USER, accumuloUsername);
-            conf.set(ConfigUtils.CLOUDBASE_PASSWORD, accumuloPassword);
-            conf.set(ConfigUtils.CLOUDBASE_INSTANCE, cluster.getInstanceName());
-            conf.set(ConfigUtils.CLOUDBASE_ZOOKEEPERS, cluster.getZooKeepers());
-            conf.set(ConfigUtils.CLOUDBASE_AUTHS, "U");
-            conf.set(ConfigUtils.USE_PCJ_FLUO_UPDATER, "true");
-            conf.set(ConfigUtils.FLUO_APP_NAME, fluoAppName);
-            conf.set(ConfigUtils.PCJ_STORAGE_TYPE, PrecomputedJoinIndexerConfig.PrecomputedJoinStorageType.ACCUMULO.toString());
-            conf.set(ConfigUtils.PCJ_UPDATER_TYPE, PrecomputedJoinIndexerConfig.PrecomputedJoinUpdaterType.FLUO.toString());
-
+            
+            AccumuloIndexingConfiguration conf = AccumuloIndexingConfiguration.builder()//
+        			.auths("U")
+        			.setAccumuloUser(accumuloUsername)
```

Comment:
Right.  Fixed my formatter to shift by spaces, but I didn't want to apply it to this class so that it would be clear what the diff was when you guys looked at my code.  I have no problem fixing the tabs now that you've had a look.